tools/run_python_system_tests.py

```diff
@@ -54,5 +56,5 @@
 print('****Running system tests in tox.****')
 tox_dir = os.path.join(zip_folder, os.listdir(zip_folder)[0])
 os.chdir(tox_dir)
-result = os.system('python -m tox -e ' + args.driver + '_system_tests')
+result = os.system('tox ' + args.python_bitness + ' -e ' + args.python_version + '-' + args.driver + '_system_tests')
```

Comment:
The seems wrong... Shouldn't -e be before args.driver?

Reply:
command should be **tox '--32' -e py36-nidmm_system_tests** for running tests on python 3.6-32,
args.python_bitness : '--32'
args.python_version : py36

Reply:
test this please